src/components/OtherHero/index.js

```diff
@@ -0,0 +1,150 @@
+import RichTypography from "@commons-ui/core/RichTypography";
+import { Grid, Typography } from "@material-ui/core";
+import { makeStyles } from "@material-ui/core/styles";
+import PropTypes from "prop-types";
+import React from "react";
+
+import mapLines from "@/pesayetu/assets/images/Mask Group 8@2x.png";
+import Section from "@/pesayetu/components/Section";
+
+const useStyles = makeStyles(({ palette, typography, breakpoints }) => ({
+  root: {
+    backgroundColor: palette.background.paper,
+    backgroundPosition: "50% 100%",
+    minHeight: "min-content",
+    position: "relative",
+    paddingBottom: typography.pxToRem(43),
+    [breakpoints.up("md")]: {
+      paddingBottom: typography.pxToRem(98),
+    },
+  },
+  textContainer: {
+    height: "100%",
+    paddingTop: typography.pxToRem(50),
+    [breakpoints.up("md")]: {
+      paddingTop: typography.pxToRem(129.94),
+    },
+  },
+  intro: {
+    fontWeight: "bold",
+  },
+  title: {
+    marginBottom: typography.pxToRem(20),
+    fontWeight: "bold",
+    [breakpoints.up("md")]: {
+      marginBottom: typography.pxToRem(40),
+    },
+  },
+  tagline: {
+    marginRight: 0,
+  },
+  backgroundGrid: {
+    width: "100%",
+    height: "100%",
+    backgroundImage: `url('${mapLines}')`,
```

Comment:
No love for `next/images`?

Reply:
erm, the map lines felt better as a background image as it didn't have to interfere the the placement of elements on the screen